src/communication/scrollsguide.py

```diff
@@ -4,10 +4,29 @@
 import aiohttp
 import json
 from urllib.parse import quote_plus
+from typing import Any, Dict, Tuple, Type
+from fuzzywuzzy import fuzz
 
+class ScrollNotFound(Exception):
+    pass
+
+
+class MultipleScrollsFound(Exception):
+    def __init__(self, scrolls, search_term):
+        self.scrolls = [scroll.name for scroll in scrolls]
+        self.search_term = search_term
+
+    def __str__(self):
+        if len(self.scrolls) > 5:
+            return f"Too many scrolls start with {self.search_term}"
```

Comment:
These two return strings should maybe say that they're doing a proper search, not just "starting with"

Reply:
Agreed